tutor/src/screens/assignment-review/index.js

```diff
@@ -122,6 +121,34 @@ class AssignmentReview extends React.Component {
 
     const Tab = AvailableTabs[this.tabIndex] || Details;
 
+    return (
+      <>
+        <StyledTabs
+          selectedIndex={this.tabIndex}
+          params={this.props.params}
+          onSelect={this.onTabSelection}
+          tabs={AvailableTabs.map(t => t.title)}
+        />
+        <Tab ux={this.ux} />
+      </>
+    );
+  }
+
+  render() {
+    const {
+      isScoresReady, course, scores, planScores, assignedPeriods, selectedPeriod, setSelectedPeriod,
+    } = this.ux;
+
+    let body;
+
+    if (selectedPeriod && !selectedPeriod.hasEnrollments) {
+      body = <MessageWrapper><NoStudentsMessage courseId={course.id} /></MessageWrapper>;
+    } else if (!isScoresReady || !scores) {
+      return <LoadingScreen message="Loading Assignment…" />;
+    } else {
+      body = this.renderTabs({ ux: this.ux });
```

Comment:
scores will be null if the period doesn't have students, so that's why this block might look strange. Also needed to wrap `NoStudentsMessage` in a wrapper since styles wouldn't attach 🤔

Reply:
Styles wouldn't attach because the `NoStudentsMessage` doesn't accept and use a `className` prop.  That's how styled-components "extends" the styles of another component, it injects a new class onto it.

Reply:
Aha! I always assumed it had a way of adding it. This is super helpful for the future, thanks!